modules/enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// 
+// 
+
+[id="enabling-workload-partitioning_{context}"]
+=  Enabling workload workload partitioning 
+
+With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition. These pods operate normally within the minimum size CPU configuration. Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
```

Comment:
Suggest to re-phrase the part about `With workload partitioning, you annotate the set of cluster management pods`
Current phrasing makes it sound like it's the end user who is supposed to add these annotations, while in reality the annotations are there as part of the Red Hat product.

Also the part about "Additional Operators and workloads" is a bit strange.
Which is the initial set of operators and workloads?